Dockerfile

```diff
@@ -0,0 +1,40 @@
+# Builds in ~ 5 min and is ~ 2.8 GB on a linux laptop
+FROM mambaorg/micromamba:0.15.3
+
+# Label image following opencontainers image-spec annotations recommendation:
+# https://github.com/opencontainers/image-spec/blob/main/annotations.md
+LABEL org.opencontainers.image.description="Container for open source time series InSAR processing with Mintpy"
+LABEL org.opencontainers.image.authors="Forrest Williams <forrestfwilliams@icloud.com>, Joseph H Kennedy <me@jhkennedy.org>, Andre Theron <andretheronsa@gmail.com>"
+LABEL org.opencontainers.image.url="https://github.com/insarlab/MintPy"
+LABEL org.opencontainers.image.source="https://github.com/insarlab/MintPy"
+LABEL org.opencontainers.image.documentation="https://mintpy.readthedocs.io/en/latest/"
+LABEL org.opencontainers.image.licenses="GPL-3.0-or-later"
+
+# Dynamic lables to define at build time via `docker build --label`
+# LABEL org.opencontainers.image.created=""
+# LABEL org.opencontainers.image.version=""
+# LABEL org.opencontainers.image.revision=""
+
+USER root
+
+ARG DEBIAN_FRONTEND=noninteractive
+ENV PYTHONDONTWRITEBYTECODE=true
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends git vim wget && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+USER micromamba
+WORKDIR /home/micromamba
+
+ENV PATH=/opt/conda/bin:${PATH}
+
+ARG MINTPY_HOME=/home/micromamba/tools/MintPy
+COPY --chown=micromamba:micromamba . ${MINTPY_HOME}/
```

Comment:
This copies MintPy into the container and will be installed into the `base` conda environment via `pip` below -- and since #659 we don't need to add the MintPy directory to `PATH`.  

Technically, the repo itself is only needed for installing, and could be removed to reduce the image size (or part of a different partial build image), but I think leaving it is nice as you can then run the tests easily.